app/pyproject.toml

```diff
@@ -7,3 +8,11 @@ module = [
     'google.*',
 ]
 ignore_missing_imports = true
+
+[tool.ruff]
+select = ["A", "B", "C", "D", "E", "F", "G", "I", "N", "Q", "S", "T", "W", "ARG", "BLE", "COM", "DJ", "DTZ", "EM", "ERA", "EXE", "FURB", "ICN", "INP", "ISC", "NPY", "PD", "PIE", "PL", "PT", "PTH", "PYI", "RET", "RSE", "RUF", "SIM", "SLF", "TCH", "TID", "TRY", "UP", "YTT"]
```

Comment:
ruff covers flake8's ruleset, but also the ruleset of 20 or so other linters, plus it's much much faster, also it can automatically fix a lot of issues. We can remove a bunch of these configs if you think it's too many rules to enforce. We can just stick to the flake8 ones maybe.

Reply:
But I think that isort and pyupgrade are really good ones to keep.

Reply:
Thanks for elaborating the context. I understand you prefer to use ruff with many rules and I understand people have different preferences on CI tooling.

However, I don't see a need to use so many lint tools for this project considering that the code base is relatively small and simple. Also, we don't have a need to use a faster lint tool. IIUC, it's difficult for a human to notice difference in the time to execute flake8 vs ruff using this project's code base.

My preference for this project is using a minimal set of tool with a minimal set of configurations. I feel the direction of this PR is not much aligned with the direction I want to go...